### ISSUE TITLE
Unable to create App engine application

### ISSUE THREAD

**Issue by cm841295** [terraform-google-modules/terraform-google-project-factory#670]
### TL;DR

I have wriiten tf files for App engine deployment,
Mentioned all the values as per terraform documentation.
Getting error as below:

Error: Error creating App Engine application: googleapi: Error 409: This application already exists and cannot be re-created., alreadyExists

### Expected behavior

Should not throw any error when deploying app engine application

### Observed behavior

google_app_engine_standard_app_version.default: Destroying... [id=apps/dev-hotstar1/services/default/versions/v1]
google_app_engine_application.app3: Creating...
google_app_engine_standard_app_version.default3: Creating...
google_app_engine_standard_app_version.default: Still destroying... [id=apps/dev-hotstar1/services/default/versions/v1, 10s elapsed]
google_app_engine_standard_app_version.default3: Still creating... [10s elapsed]
google_app_engine_standard_app_version.default: Destruction complete after 11s
google_app_engine_standard_app_version.default3: Still creating... [20s elapsed]
google_app_engine_standard_app_version.default3: Still creating... [30s elapsed]
google_app_engine_standard_app_version.default3: Still creating... [40s elapsed]
google_app_engine_standard_app_version.default3: Still creating... [50s elapsed]
google_app_engine_standard_app_version.default3: Creation complete after 53s [id=apps/dev-hotstar1/services/default-3/versions/v2]
╷
│ Error: Error creating App Engine application: googleapi: Error 409: This application already exists and cannot be re-created., alreadyExists
│ 
│   with google_app_engine_application.app3,
│   on main.tf line 1, in resource "google_app_engine_application" "app3":
│    1: resource "google_app_engine_application" "app3" {

### Terraform Configuration

```hcl
resource "google_app_engine_application" "app3" {
  project     = "dev-hotstar1"
  location_id = "us-central"
}

resource "google_app_engine_standard_app_version" "default3" {
  version_id = "v2"
  service    = "default-3"
  project    = "dev-hotstar1"
  runtime    = "go111"
  entrypoint {
    shell = "go run main.go"
  }
  deployment {
    zip {
      source_url = "https://storage.googleapis.com/dev-hotstar-001-test/sample1.zip"
    }
  }
  env_variables = {
    port = "8080"
  }
  automatic_scaling {
    max_concurrent_requests = 10
    min_idle_instances      = 1
    max_idle_instances      = 3
    min_pending_latency     = "1s"
    max_pending_latency     = "5s"
    standard_scheduler_settings {
      target_cpu_utilization        = 0.5
      target_throughput_utilization = 0.75
      min_instances                 = 2
      max_instances                 = 10
    }
  }
  delete_service_on_destroy = true
}

Note: Zip file consists of go program "Hello world"
main.go:
code below as foolows:
package main
  
import "fmt"
  
// Main function
func main() {
  
    fmt.Println("!... Hello World ...!")
}
```

### Terraform Version

```sh
terraform -v
Terraform v1.0.11
on linux_amd64
+ provider registry.terraform.io/hashicorp/google v4.5.0

Your version of Terraform is out of date! The latest version
is 1.1.2. You can update by downloading from https://www.terraform.io/downloads.html
```

### Additional information

As of now, i tried for go application,

For python application:
should i zip main.py and requirements.txt together and provide entire path in tf files?

**Comment by morgante** [comment]
Your error does not look related to this module.

Issues with the Terraform provider should be opened here: https://github.com/hashicorp/terraform-provider-google
